docs/how-to/create-a-router/configure-traffic-rules.md

```diff
@@ -8,15 +8,31 @@ This step is **optional** and the default behaviour will not discriminate betwee
 
-Traffic rules define which routes should be "activated" for a particular request to your router. Each rule is defined by one or more request conditions and one or more routes that would be activated if the request satisfies the conditions of the rule. Routes that are not part of any traffic rules will be called for each request.
+Traffic rules define which routes should be "activated" for a particular request to your router. Each rule is defined by one or more request conditions and one or more routes that would be activated if the request satisfies the conditions of the rule. 
 
-Rules are matched against the incoming request in the order in which they are defined. This property can be used to create one or more specific rules over a general rule. Consider the following example, where a router has rules defined such that requests are routed by the country of origin (ID, SG, etc.). In addition, if the routing logic must be altered for a certain service type in a country, this rule (the 'specific' rule) can be defined before the general rule.
+When traffic rules are configured, the following behaviour is expected:
+- A Default traffic rule is required, which has no request conditions and acts as a fallback rule for requests that don't match any of the configured traffic rule conditions. At least 1 route should be configured for the Default traffic rule.
+- All routes should be used by at least 1 traffic rule, i.e no dangling routes.
+- When a Nop or Standard Ensembler is configured (See: [Configure Ensembler](./configure-ensembler.md)), the final/fallback route must be present in every traffic rule, including the default rule.
+
+Rules are matched against the incoming request in the order in which they are defined. The UI supports the drag-and-drop functionality to reorder priority of the rules. This property can be used to create one or more specific rules over a general rule. Consider the following example, where a router has rules defined such that requests are routed by the country of origin (ID, SG, etc.). In addition, if the routing logic must be altered for a certain service type in a country, this rule (the 'specific' rule) can be defined before the general rule.
 
 ![](../../.gitbook/assets/create_router_rules_priority.png)
 
+In addition, orthogonality checks have been put in-place to disallow overlapping rules. An example is as described below:
+
+| Rule | Conditions |
+|------|------------|
+| 1    | country_code=[ID], geo_area=[1] |
+| 2    | country_code=[ID,SG], geo_area=[1,2,3] |
+| 3    | country_code=[SG], geo_area=[1] |
+| 4    | country_code=[ID], geo_area=[2] |
+
+From the above rules, rule 2 is overlapping with all other rules since there are no unique combinations that would let rules 3 and 4 to be called, and rule 2 will not be called for the same conditions met by rule 1.
```

Comment:
Minor comment, but it would be nice to see an example of a `valid` set of rules, kind of like a counter example to what you have shown above.

Reply:
Thanks for the example! 🙏